airflow/timetables/interval.py

```diff
@@ -79,6 +80,20 @@ class CronDataIntervalTimetable(_DataIntervalTimetable):
     def __init__(self, cron: str, timezone: datetime.tzinfo) -> None:
         self._schedule = CronSchedule(cron, timezone)
 
+    @Timetable.interval_description.getter
```

Comment:
I think just `@property` would do the same (well technically a bit different, but arguably better).